CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ...
 
 
+## [1.1.2] - 2024-11-16
```

Comment:
This should go under the `Unreleased` heading; new version headers only get added when a version actually gets published. The brackets also indicate a markdown link with a reference defined later; in this case, the comparison link for this would need to be added at the bottom of the file.